### PR TITLE
Tron 10

### PR DESCRIPTION
**Pull request by Hassan0112**
Ticket:https://pakw.atlassian.net/browse/TRON-10

- **Why?**
- 
- Now we want to create a new form for the users where they can post their adds for their used-cars to sell them out.
- 
- **Acceptance Criteria:**
- 
- When User clicks on “Post an Ad” button on used-cars index page specifically on “Post your Ad on PakWheels” card section, it should take user to a new page where user can see a form to fill in details about their Ad they wanted to post out.
- 
- Details about the page:
- 
- Navbar header will render the same way it renders on every single page of the application.
- 
- Under that add a section to describe user to post their add with 3 simple steps (Design attached).
- 
- Add another section where user can add the information about their vehicle (car_info, city color etc..) (Design attached).
- 
- Add another section where user can add multiple pictures of their vehicle(Design attached).
- 
- Add another section where user can add the contact information (e.g phone No.) (Design attached)
- 
- At the end Add a button to submit the form.

### CODE REVIEW

File: app/javascript/packs/new_ad.js [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+
+      // Display the converted result
+
+      $('.para5-text').text( result);
+    
+    } 
+  });
+});
+
+function convertNumber() {
```
Comment (Adil449) [inline app/javascript/packs/new_ad.js:71]: Where this function is called?

File: app/javascript/packs/new_ad.js [hunk 2]
```diff
@@ -0,0 +1,137 @@
+$(document).ready(function() {
```
Comment (Adil449) [inline app/javascript/packs/new_ad.js:1]: do not use multiple `$(document).ready(function()` keep everything under a single `$(document).ready(function()` wherever it is being used

File: config/routes.rb [hunk 3]
```diff
@@ -7,5 +7,7 @@
   patch '/profile', to: 'users#update'
   get '/profile', to: 'users#profile', as: 'user_profile'
   patch '/update_profile_picture', to: 'users#update_profile_picture', as: 'update_profile_picture'
+  get 'used-cars/sell', to: 'used_cars#index'
```
Comment (Adil449) [inline config/routes.rb:10]: This route was already added in Tron-9 ticket here is the pull request for that =>https://github.com/Hassan0112/Trons/pull/10/files#diff-959bc9abc46a55332bb64d5155a79323afa75a50ec1a2137ddd22d926f62c6c5, why that change is also reflecting into this PR

File: config/routes.rb [hunk 4]
```diff
@@ -7,5 +7,7 @@
...
+  get '/post-ad' , to: 'used_cars#new_ad'
```
Comment (Adil449) [inline config/routes.rb:11]: This route should be `used-cars/sell, to: 'used_cars#post-ad'` the way we have here on actual domain => https://www.pakwheels.com/used-cars/sell/post-ad

File: app/javascript/packs/new_ad.js (hunk 1 above)
Comment (Hassan0112) [reply app/javascript/packs/new_ad.js:71]: i remove it

Comment (Adil449) [reply app/javascript/packs/new_ad.js:71]: Alright, make a another commit for that and push it

**Comment by Adil449** [comment]
Resolve Conflicts

**Review by Adil449 — APPROVED** [review]

**Pull request merged**